tagstore/tagstore_rest/v1/views/entities.py

```diff
@@ -62,5 +67,69 @@ def tags(self, request, entity_type=None, pk=None):
             queryset = entity.tags.all()
 
         extracted = [self._convert(tag) for tag in queryset]
-        serializer = EntityTagSerializer({'tags': extracted})
+        return EntityTagSerializer({'tags': extracted})
+
+    def tags(self, request, entity_type=None, pk=None):
+        '''
+        Get a list of all tags belonging to an entity.
+
+        These can be optionally filtered by `taxonomy` params.
+        '''
+        entity = get_object_or_404(Entity, external_id=pk, entity_type=entity_type)
+        taxonomies = request.GET.get('taxonomies', '').split(',')
+
+        serializer = self.serialize_tags(entity, taxonomies)
         return Response(serializer.data)
+
+    def update_tags(self, request, entity_type=None, pk=None):
+        '''
+        Update tags belonging to an entity.
+        '''
+        tagstore = DjangoTagstore()
+
+        entity_obj = get_object_or_404(Entity, external_id=pk, entity_type=entity_type)
+        entity = EntityId(external_id=pk, entity_type=entity_type)
```

Comment:
Nit: You could just use `entity_obj.as_tuple` in the two places `entity` is used, without needing this line to initialize it.